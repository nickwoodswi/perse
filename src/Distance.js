import React, { Component } from 'react'

class Distance extends Component {
    render() {
        if (!this.props.distance) {
            return(
                <></>
            )
        }
        else {
            return(
                <> {this.props.distance}m</>
            )
        }
    }
}

export default Distance