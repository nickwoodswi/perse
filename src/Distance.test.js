import React from 'react';
import { render } from '@testing-library/react';
import Distance from './Distance';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

test('renders Distance component', () => {
    render(<MemoryRouter><Distance /></MemoryRouter>);
});

it('Distance renders the UI as expected', () => {
    const tree = renderer
        .create(<MemoryRouter><Distance /></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});