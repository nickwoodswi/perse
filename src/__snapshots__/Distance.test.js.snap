// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Distance renders the UI as expected 1`] = `null`;
